examples/inventory-test-alpine.yml

```diff
@@ -0,0 +1,61 @@
+all:
```

Comment:
You forgot to add the alpine test on Travis

Reply:
Remember also to push the image on the cloud to be pulled during the pipeline

Reply:
Added